src/Style/Sass/component/_navLogo.scss

```diff
@@ -27,3 +27,19 @@
     font-size: 20px;
   }
 }
+
+#web-name{
+  margin: 10px;
+  font-family: 'Times New Roman', Times, serif;
```

Comment:
explain this font?

Reply:
that one is used in the logo, you can go there and check it looks good in my opinion